Tests.Integration.Presentation.Web/ItSystem/ItSystemUsageMigrationTests.cs

```diff
@@ -0,0 +1,82 @@
+        {
+            //Arrange
+            var itSystemName = CreateName();
+            var itSystem = await ItSystemHelper.CreateItSystemInInitialOrganizationAsync(itSystemName, 1);
+
+            var token = await HttpApi.GetTokenAsync(role);
+            var orgId = 1;
+            var url = TestEnvironment.CreateUrl($"api/v1/ItSystemUsageMigration/UnusedItSystems" +
+                                                $"?organizationId={orgId}" +
+                                                $"&nameContent={itSystemName}" +
+                                                $"&numberOfItSystems={10}" +
+                                                $"&getPublicFromOtherOrganizations={true}");
+
+            //Act
+            using (var httpResponse = await HttpApi.GetWithTokenAsync(url, token.Token))
+            {
+                var response = await httpResponse.ReadResponseBodyAsKitosApiResponseAsync<IEnumerable<Core.DomainModel.ItSystem.ItSystem>>();
+                //Assert
+                Assert.Equal(HttpStatusCode.OK, httpResponse.StatusCode);
+                var itSystems = response.ToList();
+                Assert.NotEmpty(itSystems);
+                Assert.Equal(1, itSystems.Count(x => x.Name == itSystemName));
+            }
+        }
+
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        [InlineData(OrganizationRole.User)]
+        public async Task Api_User_Can_Limit_How_Many_Systems_To_Return(OrganizationRole role)
+        {
+            //Arrange
+            var prefix = "migrationTestSystem";
+            var itSystemName1 = prefix + CreateName();
+            var itSystemName2 = prefix + CreateName();
+            var itSystem1 = await ItSystemHelper.CreateItSystemInInitialOrganizationAsync(itSystemName1, 1);
+            var itSystem2 = await ItSystemHelper.CreateItSystemInInitialOrganizationAsync(itSystemName2, 1);
+
+            var token = await HttpApi.GetTokenAsync(role);
+            var orgId = 1;
+            var url = TestEnvironment.CreateUrl($"api/v1/ItSystemUsageMigration/UnusedItSystems" +
+                                                $"?organizationId={orgId}" +
+                                                $"&nameContent={prefix}" +
+                                                $"&numberOfItSystems={2}" +
+                                                $"&getPublicFromOtherOrganizations={true}");
+
+            //Act
+            using (var httpResponse = await HttpApi.GetWithTokenAsync(url, token.Token))
+            {
+                var response = await httpResponse.ReadResponseBodyAsKitosApiResponseAsync<IEnumerable<Core.DomainModel.ItSystem.ItSystem>>();
+                //Assert
+                Assert.Equal(HttpStatusCode.OK, httpResponse.StatusCode);
+                var itSystems = response.ToList();
+                Assert.NotEmpty(itSystems);
+                Assert.Equal(2, itSystems.Count);
+            }
+        }
+
+
```

Comment:
Der mangler en test der viser at man kun får public fra andre organisationer end ens egen. Der må du bruge global admin til at oprette systemer i begge og så almindelig bruger til at hente dem ud (unikt prefix igen for at sikre forudsætningerne) og så validere at kun dem der opfyldte din betingelse kom med